core/templates/dev/head/components/forms/image_uploader_directive.html

```diff
@@ -10,6 +10,7 @@
     Please do not upload watermarked images.
   </span>
   <label for="image-file-input" translate="I18N_DIRECTIVES_UPLOAD_A_FILE" class="image-uploader-upload-label-button"></label>
+  <!--onclick="this.value=null" resets the value of the input so that uploading the same image twice in a row works.-->
```

Comment:
Some minor nits:

- Put a space after the initial `<!--`, for readability.
- Put a space before the terminal `-->`, for readability.
- Mention that this is only an issue on Chrome.
- Provide a link to a reference (e.g. the shortlink for a StackOverflow post).
- Perhaps change the first part to `The onclick="this.value=null" attribute` to help set context better, otherwise it sort of looks like commented-out code.